example-external-task-process/src/test/java/info/novatec/micronaut/camunda/bpm/externaltask/process/ProcessTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package info.novatec.micronaut.camunda.bpm.externaltask.process;
+
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.camunda.bpm.engine.RuntimeService;
+import org.camunda.bpm.engine.runtime.ProcessInstance;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+
+import static org.camunda.bpm.engine.test.assertions.bpmn.BpmnAwareTests.assertThat;
+import static org.camunda.bpm.engine.test.assertions.bpmn.BpmnAwareTests.complete;
+import static org.camunda.bpm.engine.test.assertions.bpmn.BpmnAwareTests.externalTask;
+import static org.camunda.bpm.engine.test.assertions.bpmn.BpmnAwareTests.withVariables;
+
+@MicronautTest
+public class ProcessTest {
+
+    @Inject
+    private RuntimeService runtimeService;
+
+    @Test
+    public void externalTaskTest() {
+
+        ProcessInstance processInstance = runtimeService.startProcessInstanceByKey("externalTaskProcessExample");
+        assertThat(processInstance).isStarted();
+
+        assertThat(processInstance).isWaitingAt("externalTask").externalTask().hasTopicName("item");
```

Comment:
lets rename the topic from "item" to "my-topic"

Reply:
I'd like to have "topic" in the name of the topic to make it clear that it is a topic. That is why I proposed "my-topic" :-)